hive/assets/service_checks.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "agent_version": "6.13.0",
+        "integration": "Hive",
+        "groups": ["host", "instance"],
+        "check": "hive.can_connect",
+        "statuses": ["ok", "critical"],
+        "name": "Can Connect",
+        "description": "Returns `CRITICAL` if the Agent is unable to connect to and collect metrics from the monitored HiveServer2/Hive Metastore instance. Returns `OK` otherwise."
```

Comment:
```suggestion
        "description": "Returns `CRITICAL` if the Agent is unable to connect to and collect metrics from the monitored HiveServer2/Hive Metastore instance, otherwise returns `OK`."
```